### ISSUE TITLE
train script

### ISSUE THREAD

**Issue by andriyrizhiy** [CVBase-Bupt/EndtoEndCroppingSystem#2]
How to train your model? where training script?

**Comment by andriyrizhiy** [comment]
Because when i trying to train all model, i have en error `An operation has `None` for gradient. Please make sure that all of your ops have a gradient defined (i.e. are differentiable). Common ops without gradient: K.argmax, K.round, K.eval.`. Gradient = None after RoiPoolingConv

**Comment by joehalfish** [comment]
Would you like to tell me how to train the model ? i'm a newer ,it uses the pre-train and i want to do it myself. thanks
